website/views/shared/modals/members.jade

```diff
@@ -46,9 +46,12 @@ script(type='text/ng-template', id='modals/private-message.html')
   .modal-header
     h4=env.t('pmHeading', {name: "{{profile.profile.name}}"})
   .modal-body
-    textarea.form-control(type='text',rows='5',ui-keydown='{"meta-enter":"sendPrivateMessage(profile._id, _message)"}',ng-model='_message')
+    textarea.form-control(type='text',rows='5',ui-keydown='{"meta-enter":"sendPrivateMessage(profile._id, _message)"}',placeholder=env.t('needsTextPlaceholder'),ng-model='_message')
   .modal-footer
-    button.btn.btn-primary(ng-click='sendPrivateMessage(profile._id, _message)')=env.t("send")
+    //- Due to a quirk in Bootstrap UI, we need to wrap this so the tooltip can
+    //- appear when disabled.
+    div(style='display:inline-block',tooltip="{{_message ? '' : env.t('needsText')}}")
```

Comment:
I did some research, and using a wrapper div with the tooltip is actually the recommended way of handling our situation in the Bootstrap 3 documents themselves. Link embedded into the comments for clarity/explanation, since it's still a little on the hacky side. Still, Bootstrap themselves recommend it, sooooo.

I tried using .visible-xs-inline-block, and it just ended up displaying as a normal block element. It's possible I'm using it incorrectly, though. In the meantime, I created a compromise of putting "display: inline-block" in a separate class inside the CSS. That way, it's at least out of the style attribute, and we can reuse it if we ever need to do this again :)